tests/reference/runtime-test_list_01-3ee9b3e.stderr

```diff
@@ -1 +1 @@
-ValueError: The list does not contain the element
+sh: 1: ./a.out: Text file busy
```

Comment:
This happens when I run `./run_tests,py -u`.
So this is happening because of: https://stackoverflow.com/questions/16764946/what-generates-the-text-file-busy-message-in-unix
So, I think for every file compilation we should generate a unique executable (but not `a.out`). And after printing the output, I think we should delete the executable?
cc @certik What are your thoughts?

Reply:
This issue was fixed by writing the executable into a new unique file, say `filename.out`.